src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -96,49 +109,64 @@ const SetAclPermissionForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
-  /* eslint-disable jsx-a11y/label-has-associated-control */
   return (
     <FormSection
       title="Permission to Files"
       state={state}
       statusType="Permission status"
       defaultMessage="To be set..."
     >
-      <form onSubmit={handleAclPermission} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="set-acl-to">Set permissions to username: </label>
-          <br />
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleAclPermission} autoComplete="off">
+          <FormControl>
+            <Typography htmlFor="set-acl-to">Set permissions to username:</Typography>
+            <TextField
+              id="set-acl-to"
+              name="setAclTo"
+              {...username}
+              placeholder={selectedUser}
+              label="Search username"
+              required
+            />
+          </FormControl>
+          <InputLabel htmlFor="set-acl-doctype">
+            <em>Select Document Type</em>
+          </InputLabel>
+          <FormControl fullWidth>
+            <DocumentSelection htmlId="set-acl-doctype" />
+          </FormControl>
           <br />
-          <input
-            id="set-acl-to"
-            size="25"
-            name="setAclTo"
-            {...username}
-            placeholder={selectedUser}
-          />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="set-acl-doctype">Select document type: </label>
-          <DocumentSelection htmlId="set-acl-doctype" />{' '}
-        </div>
-        <div style={formRowStyle}>
-          <p>Select permission setting:</p>
-          <input type="radio" id="set-acl-perm-give" name="setAclPerms" value="Give" />
-          <label htmlFor="set-acl-perm-give">Give</label>
-          <input type="radio" id="set-acl-perm-revoke" name="setAclPerms" value="Revoke" />
-          <label htmlFor="set-acl-perm-revoke">Revoke</label>
-        </div>
-        <button disabled={state.processing} type="submit">
-          Set Permission
-        </button>
-      </form>
+          <FormControl fullWidth>
+            <FormLabel htmlFor="set-acl-perm-label">Select permission setting:</FormLabel>
```

Comment:
Google console issues says red page error:

"Incorrect use of `<label for=FORM_ELEMENT>`
The label's for attribute doesn't match any element id."

MUI docs says the `htmlFor` should be `id` (like in `SetAclPermsDocContainerForm`).

---

But when it is `id` it says blue improvement issue:

"No label associated with a form field
A `<label>` isn't associated with a form field.
To fix this issue, nest the `<input>` in the `<label>` or provide a for attribute on the `<label>` that matches a form field id."

---

I guess go with `id` like other component and MUI docs say.  A blue warning is probably better than a red error. 😆

Reply:
Looking at all of the violating nodes for no label associated in /PASS/forms, it's likely due to DocumentSelection.jsx still being HTML elements instead of MUI components.